<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>551</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Play">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Pause">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Bw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Fw">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Vdown">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>V-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Vup">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>V+</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
